test/parallel/test-fs-watch.js

```diff
@@ -64,6 +64,8 @@ for (const testCase of cases) {
       assert.strictEqual(eventType, 'change');
     assert.strictEqual(argFilename, testCase.fileName);
 
+    watcher.start(); // should not crash
```

Comment:
Doesn't the linter complain about using two spaces before `//`?

Reply:
No it doesn't, though fixed anyway.